package.json

```diff
@@ -32,7 +32,7 @@
     "node"
   ],
   "author": "Sam CH Cho",
-  "license": "ISC",
```

Comment:
The ISC license was probably generated by npm when initializing the project?

Reply:
:)